packages/edit-site/src/components/left-sidebar/navigation-panel/style.scss

```diff
@@ -8,6 +8,15 @@
 	}
 }
 
+.edit-site-navigation-panel__back-to-dashboard.components-button.is-tertiary {
+	border-radius: 0;
+	border-bottom: 1px solid $gray-700;
```

Comment:
nitpicking: `$border-width` instead of `1px` ? 😄

Reply:
💥